docs/advanced-entity-analytics/entity-store.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[entity-store]]
+= Entity store
+
+preview::[]
+
+.Requirements
+[sidebar]
+--
+To use the entity store, you must have the appropriate privileges. For more information, refer to <<ers-requirements, Entity risk scoring requirements>>.
+--
+
+The entity store allows you to query, reconcile, maintain, and persist entity metadata from various sources such as:
+
+* Ingested log sources
+* Integrated identity providers (such as Active Directory, EntraID, and Okta)
+* Internal and external alerts
+* External asset repositories
+* Asset criticality submissions
```

Comment:
```suggestion
The entity store allows you to query, reconcile, maintain, and persist entity metadata such as:

* Ingested logs
* Integrated identity data (such as from Active Directory, EntraID, and Okta)
* Internal and external alerts
* External asset repository data
* Asset criticality data
```
Just another option that occurred to me

Reply:
We can also add "Entity risk score data" to this list